kong/clustering/rpc/manager.lua

```diff
@@ -253,31 +298,22 @@ function _M:handle_websocket()
     return ngx_exit(ngx.HTTP_CLOSE)
   end
 
-  if rpc_protocol ~= "kong.rpc.v1" then
+  -- TODO: choice a proper protocol
+  if not rpc_protocol:find(RPC_MATA_V1, 1, true) then
```

Comment:
We need to properly parse the list, if `Sec-WebSocket-Protocol: foo.kong.meta.v1` then this is not compatible and connection must be closed as well.

Reply:
done.